schemas/submission_v1.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$id": "submission_v1.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "ONS EQ Submission Schema v1",
+  "properties": {
+    "case_id": {
+      "$ref": "common/definitions.json#/case_id"
+    },
+    "case_ref": {
+      "$ref": "common/definitions.json#/case_ref"
+    },
+    "case_type": {
+      "$ref": "common/definitions.json#/case_type"
+    },
+    "channel": {
+      "$ref": "common/definitions.json#/channel"
+    },
+    "collection": {
+      "additionalProperties": false,
+      "properties": {
+        "exercise_sid": {
+          "$ref": "common/definitions.json#/collection_exercise_sid"
+        },
+        "instrument_id": {
+          "$ref": "common/definitions.json#/form_type"
+        },
+        "period": {
+          "$ref": "common/definitions.json#/period_id"
+        },
+        "schema_name": {
+          "$ref": "common/definitions.json#/schema_name"
+        }
+      },
+      "required": ["exercise_sid", "period", "schema_name", "instrument_id"],
```

Comment:
schema name is not provided by EQv2 and so should not be required

Reply:
Done